app/static/js/events/wizard/vue-app.js

```diff
@@ -387,6 +393,7 @@ function shouldDisableMove($this) {
 
 function checkTickets($this) {
     var flag = false;
+    if($this !== null)
     $this.forEach(function(ticket){
         if(ticket.type === 'paid' && (ticket.price === '' || ticket.price <=0))
         {
```

Comment:
Please fix this indentation as well. I know maybe it's not your fault. But please refactor this.